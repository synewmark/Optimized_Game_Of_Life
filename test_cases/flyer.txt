version https://git-lfs.github.com/spec/v1
oid sha256:ea86251d487efe1cb43ccbab8123c18f092d2f057d01b52c8b01f83daaeea13e
size 1002020
