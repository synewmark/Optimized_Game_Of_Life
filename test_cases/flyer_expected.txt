version https://git-lfs.github.com/spec/v1
oid sha256:6134381f58a5bdbc10f5d64b14544b46ffd87a6e79554aca9da8cdd0f7ab8fbb
size 1001000
